.changeset/lazy-turtles-bread.md

```diff
@@ -0,0 +1,7 @@
+---
+'@xstate/test': major
+---
+
+@author: @davidkpiano
+
+Removed `.testCoverage()`, and instead made `getPlans`, `getShortestPlans` and `getSimplePlans` cover all paths by default.
```

Comment:
Just being more accurate - it's still possible for some transitions not to be covered because of non-exhaustive event cases

```suggestion
Removed `.testCoverage()`, and instead made `getPlans`, `getShortestPlans` and `getSimplePlans` cover all states and transitions enabled by event cases by default.
```